### ISSUE TITLE
Make Marketplace a local (not an external) link

### ISSUE THREAD

**Issue by mark-a-wilson** [bcgov/digital.gov.bc.ca#902]
Per the [GDX accessibility audit](https://bcgov.sharepoint.com/:b:/t/00109-digital.gov.bc.ca/EQrwcYPvVqNPrpsUS_cmXPMBtgckjCybC3Nm02RqwMbRLQ?e=Jqz9Da), the Digital Marketplace should be considered to be the same site, not an external site. 

#### Acceptance Criteria

- [x] Remove external link icon beside the Marketplace link and make the link open the Marketplace app in the same window.

**Comment by SteveChapmanBCDX** [comment]
Reviewed with @mark-a-wilson - signing off ticket.
